docs/user/tutorials/00-25-scheduling-best-practices.md

```diff
@@ -0,0 +1,11 @@
+## Scheduling Best Practices
+
```

Comment:
We need an introductory paragraph here: what the document is about and what to use it for. For example:
```suggestion

> [!WARNING]
> NFS backup scheduling is a beta feature available only per request for SAP-internal teams.

Using the NFS backup scheduling, you can create backups at specified intervals. To use the feature effectively, follow the scheduling best practices: 
```